### DISCUSSION TITLE
Proxmox + Immich + OMV. Change Upload location

### DISCUSSION THREAD

**Discussion by eemenda** [immich-app/immich#14201] [Q&A] [answered]
Hi.
I have a Proxmox setup with OpenMediaVault in a VM and Immich in a container with Docker.
In OMV, I share a folder via SambaShare and mount it in Proxmox. I add the mount point to the 107.conf file of the container so it becomes available.
According to the documentation, in the Immich installation folder, I modify the .env file. Where it says:
UPLOAD_LOCATION=./library,
I replace it with:
UPLOAD_LOCATION=/mnt/disk.
I restart the container to apply the changes. It starts, but I can no longer access its IP over the network.
To check the permissions, I manually created a folder inside the mount point /mnt/disk from the Immich console, and I was able to create it.

What am I doing wrong?

**Comment by bo0tzz** [comment]
Did you look at the logs from the Immich container? When you changed the path did you also move the files over?

**Reply by eemenda** [reply]
It's my first upload. I haven't files yet
I'm going to see the logs
But in fact, no matter the new location, when I modify the location upload in the env file, i lost ip acces

**Reply by eemenda** [reply, marked as answer]
Resolved:  
I had to modify the `.env` file but also copy the folder structure from the library to the new location.
